src/components/calltree/FilterNavigatorBar.css

```diff
@@ -71,48 +71,37 @@
   border-bottom-color: transparent !important;
 }
 
-.filterNavigatorBarItem:not(.filterNavigatorBarLeafItem):not(.filterNavigatorBarBeforeSelectedItem):not(.filterNavigatorBarSelectedItem)::after {
+.filterNavigatorBarItem:not(.filterNavigatorBarLeafItem)::after {
   background-image: url(../../../res/scope-bar-separator.svg);
   background-repeat: no-repeat;
   background-position: -18px -12px;
   background-size: 24px 24px;
 }
 
 .filterNavigatorBarSelectedItem {
-  background-color: var(--internal-selected-color);
-  color: white;
+  color: var(--internal-selected-color);
 }
 
-.filterNavigatorBarSelectedItem:not(.filterNavigatorBarRootItem)::before,
-.filterNavigatorBarSelectedItem.filterNavigatorBarRootItem,
-.filterNavigatorBarSelectedItem:not(.filterNavigatorBarLeafItem)::after {
-  border-color: var(--internal-selected-color);
-}
-
-.filterNavigatorBarItem:not(.filterNavigatorBarSelectedItem):hover {
+.filterNavigatorBarItem:not(.filterNavigatorBarLeafItem):hover {
   background-color: rgba(0, 0, 0, 0.1);
 }
 
-.filterNavigatorBarItem:not(.filterNavigatorBarSelectedItem):not(.filterNavigatorBarRootItem):hover::before,
-.filterNavigatorBarItem:not(.filterNavigatorBarSelectedItem).filterNavigatorBarRootItem:hover,
-.filterNavigatorBarItem:not(.filterNavigatorBarSelectedItem):not(.filterNavigatorBarLeafItem):hover::after {
+.filterNavigatorBarItem:not(.filterNavigatorBarRootItem):not(.filterNavigatorBarLeafItem):hover::before,
+.filterNavigatorBarItem.filterNavigatorBarRootItem:hover,
+.filterNavigatorBarItem:not(.filterNavigatorBarLeafItem):hover::after {
   border-color: rgba(0, 0, 0, 0.1);
 }
 
-.filterNavigatorBarItem:not(.filterNavigatorBarSelectedItem):active:hover {
+.filterNavigatorBarItem:not(.filterNavigatorBarLeafItem):active:hover {
   background-color: rgba(0, 0, 0, 0.2);
 }
 
-.filterNavigatorBarItem:not(.filterNavigatorBarSelectedItem):not(.filterNavigatorBarRootItem):active:hover::before,
-.filterNavigatorBarItem:not(.filterNavigatorBarSelectedItem).filterNavigatorBarRootItem:active:hover,
-.filterNavigatorBarItem:not(.filterNavigatorBarSelectedItem):not(.filterNavigatorBarLeafItem):active:hover::after {
+.filterNavigatorBarItem:not(.filterNavigatorBarRootItem):not(.filterNavigatorBarLeafItem):active:hover::before,
+.filterNavigatorBarItem.filterNavigatorBarRootItem:active:hover,
+.filterNavigatorBarItem:not(.filterNavigatorBarLeafItem):active:hover::after {
```

Comment:
not for this patch, but I think we use `:not` too much. This makes our CSS difficult to read, difficult to maintain as it adds specificity, and I'm pretty sure this is not very performant although that's the least of my concerns for now.